website/source/docs/providers/aws/r/kinesis_firehose_delivery_stream.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "aws"
+page_title: "AWS: aws_kinesis_firehose_delivery_stream"
+sidebar_current: "docs-aws-resource-kinesis-firehose-delivery-stream"
+description: |-
+  Provides a AWS Kinesis Firehose Delivery Stream
+---
+
+# aws\_kinesis\_stream
+
+Provides a Kinesis Firehose Delivery Stream resource. Amazon Kinesis Firehose is a fully managed, elastic service to easily deliver real-time data streams to destinations such as Amazon S3 and Amazon Redshift.
+
+For more details, see the [Amazon Kinesis Firehose Documentation][1].
+
+## Example Usage
+
+```
+resource "aws_s3_bucket" "bucket" {
+	bucket = "tf-test-bucket"
+	acl = "private"
+}
+
+esource "aws_iam_role" "firehose_role" {
+   name = "firehose_test_role"
+   assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "firehose.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_kinesis_firehose_delivery_stream" "test_stream" {
+	name = "terraform-kinesis-firehose-test-stream"
+	destination = "s3"
+	role_arn = "${aws_iam_role.firehose_role.arn}"
+	s3_bucket_arn = "${aws_s3_bucket.bucket.arn}"
+}
+```
+
+~> **NOTE:** Kinesis Firehose is currently only support in us-east-1, us-west-2 and eu-west-1. This implementation of Kinesis Firehose only supports the s3 destination type 
```

Comment:
There's a minor typo here: support -> supported. It might also be worth calling out explicity that this resource only supports S3 as Terraform doesn't yet support Redshift.